articles/manual/statements.md

```diff
@@ -1,14 +1,16 @@
 ---
 title: Q# statements | Microsoft Docs 
-description: Q# statements
+description: Q# statements, Control Flow and other constructs
 author: QuantumWriter
-uid: microsoft.quantum.qsharp-ref.statements
+uid: microsoft.quantum.manual.statements
 ms.author: Alan.Geller@microsoft.com 
 ms.date: 12/11/2017
 ms.topic: article
 ---
 
-# Statements and Other Constructs
+# Statements, Control Flow and other constructs 
+
+This section contains description of statements,work flow elements like loops or conditonals and other kind of constructs.
```

Comment:
"workflow elements" reads a bit awkwardly to me, but I'm not sure what to suggest instead, sorry.

Reply:
How about "control flow structures"?